### PR TITLE
[CAZB-2999] Update PaymentsApi to use cachedCharge from AccountsApi

### PR DESCRIPTION
**Pull request by dratka**
JiraCard: https://eaflood.atlassian.net/browse/CAZB-2999
Partent: https://eaflood.atlassian.net/browse/CAZB-2948

Updated the GET `/accoutns/:account_id/charges` endpoint to use cachedCharge retuned by the Accounts API.

### CODE REVIEW

File: src/main/java/uk/gov/caz/psr/dto/CachedCharge.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import lombok.Builder;
+import lombok.Value;
+
+/**
+ * Class that represents the JSON structure for vehicle retrieval response for vehicles and charges
+ * and stores cached charge.
+ */
+@Value
+@Builder
+public class CachedCharge {
```
Comment (plenza) [inline src/main/java/uk/gov/caz/psr/dto/CachedCharge.java:13]: @radekmorytko @dratka  - Guys - what do you think about moving these common Account&Payment DTOs into Common https://github.com/InformedSolutions/JAQU-CAZ-Java-Common/tree/develop/definitions project?

I can easily be persuaded against by using "there is no time" argument :)

**Review by plenza — APPROVED** [review]

File: src/main/java/uk/gov/caz/psr/dto/CachedCharge.java (hunk 1 above)
Comment (radekmorytko) [reply src/main/java/uk/gov/caz/psr/dto/CachedCharge.java:13]: I agree, PR: https://github.com/InformedSolutions/JAQU-CAZ-Java-Common/pull/214/files

**Comment by cruddasj** [comment]
@radekmorytko, @dratka, @plenza - FYI the single vehicle chargeability piece has been merged up to ST, so this should now be good to merge down.

**Review by plenza — APPROVED** [review]

**Comment by radekmorytko** [comment]
Good change :+1: 
image

**Review by radekmorytko — APPROVED** [review]

**Pull request merged**